test/testGraphical.m

```diff
@@ -0,0 +1,47 @@
+function [ status ] = testGraphical( varargin )
+%TESTGRAPHICAL Runs the M2T test suite to produce graphical output
+%
+% This is quite a thin wrapper around testMatlab2tikz to run the test suite to
+% produce a PDF side-by-side report.
+%
+% Its allowed arguments are the same as those of testMatlab2tikz.
+%
+% Usage:
+%
+%     status = testGraphical(...) % gives programmatical access to the data
+%
+%     testGraphical(...); % automatically invokes makeLatexReport afterwards
+%
+% See also: testMatlab2tikz, testHeadless, makeLatexReport
+
+    cwd = initializeWorkingDirectory();
```

Comment:
I meant -- we still struggle with different settings in `get(0)` that (can) influence the test results. Before starting to work on the actual code, we should have a common understanding of where to intervene. I think after setting up the working directory and before starting the actual tests is the correct place. We could then restore the user's default settings `cGraphicalSettings` (c = current) right after the execution of `testMatlab2tikz` in the line below.